pubspec.yaml

```diff
@@ -1,16 +1,16 @@
 name: graphs
-version: 0.2.1-dev
+version: 0.3.0-nullsafety.0
```

Comment:
just make this 0.3.0 – we're ready to go stable w/ these

Reply:
done